corehq/apps/es/tests/test_migration_operations.py

```diff
@@ -271,7 +271,7 @@ def test_deconstruct_omits_mapping__meta(self):
             "top_level": True,
             "_meta": {"key": "value"},
         }
-        operation = CreateIndex(name, type_, mapping, analysis, settings_key)
+        operation = CreateIndex(name, type_, mapping, analysis, settings_key, es_versions=[])
```

Comment:
Is this one necessary since the __init__ method for `CreateIndex` should default to `es_versions=[]`?

Reply:
Good catch. I remember changing this test first for checking version awareness, then ended up writing a separate and the param was not removed.

Reply:
https://github.com/dimagi/commcare-hq/pull/33468/commits/b8297a3c2a32208967ee1eefa3bab27152f3efec